src/main/java/com/amazon/dlic/auth/http/saml/HTTPSamlAuthenticator.java

```diff
@@ -73,6 +75,15 @@
     public static final String IDP_METADATA_FILE = "idp.metadata_file";
     public static final String IDP_METADATA_CONTENT = "idp.metadata_content";
 
+    private static final String LEGACY_OPENDISTRO_PREFIX = "/_opendistro/_security/";
+    private static final String PLUGINS_PREFIX = "/_plugins/_security/";
+    private static final String API_AUTHTOKEN_SUFFIX = "api/authtoken";
```

Comment:
We should use some utility for such variable, seems very generic and should not be here

Reply:
This to be used for compiled patterns and it was based on Vlad's suggestion.

Reply:
Azar,  my suggestion was to use compiled pattern and avoid 2 similar string comparison. Palash point is that both `"/_opendistro/_security/"` and `"/_plugins/_security/"` are not specific to this class and should be moved to a different (common) utility class. Please take that as an action item for follow up PR.

Reply:
Yes Vlad, I talked to Palash offline as well.
Sure, I'll fix all of this in another PR.